### ISSUE TITLE
Send diagnostics tool info to Slack

### ISSUE THREAD

**Issue by thebiglabasky** [metabase/metabase#49542]
Whenever clicking the "Report bug" button (cf. #49543) we should send to Slack a summary message along with the JSON file currently generated by the diagnostics tool.

**Comment by filiphric** [comment]
update: the PR Is open, but I need to do some more adjustments to it. We discussed my approach with Vamsi. I chose to add another module so that bugs are reported to our slack only. The thing is, that we already have an existing slack integration that allows people to integrate slack server of their choice. My assumption was that we only want to have bug reports sent to us, but Vamsi pointed out, that there’s a strong argument for our users to be able to set up an internal bug reporting channel for themselves and forward bug reports to us when needed. I’m going to adjust the diagnostic tool for that and open PR

**Comment by filiphric** [comment]
UPDATE: most of the logic for this was done today. Just making finishing touches to the slack message formatting. After that, I’ll make PR, and fix tests if needed.

**Comment by thebiglabasky** [comment]
Nice. The Slack argument is fair, though we will push bugs primarily to Pylon and GitHub for end users since not all of them use Slack (some MS Teams) and not all of them have a shared channel with us. So we'll start with what everyone can use. Now all that is subsequent to this initial Slack task which is for us only so that's all fine 👌🏻

**Comment by filiphric** [comment]
Created a [new PR](https://github.com/metabase/metabase/pull/49875) for this issue. the feature is full working and needs `MB_BUG_REPORTER_ENABLED` env variable. 

![Image](https://github.com/user-attachments/assets/8aacb037-9c9e-4fea-82d1-ecebcacc4fd2)
![Image](https://github.com/user-attachments/assets/21080e82-2c67-4a6b-8314-dc1756892173)
![Image](https://github.com/user-attachments/assets/129ac6bd-6157-4a8d-8dc9-ecfd6ef50eb9)

**Comment by filiphric** [comment]
UPDATE: there were indeed some failed tests, lint and type errors. as of now, they are all fixed and PR is waiting for review.

**Comment by filiphric** [comment]
update: some requests on the implementation logic were requested by fellow developers. these have now been addressed and PR was updated. I haven’t got any feedback from backend developers yet.

**Comment by thebiglabasky** [comment]
Okay, let me know if you need help to push this a little forward and unblock the rest
